### PR TITLE
Expose ship class base stats

### PR DESCRIPTION
**Pull request by twof**
Useful for various physics calculations

Accessible in game via

```rust
class().default_stats()
```

### CODE REVIEW

File: shared/api/src/lib.rs [hunk 1]
```diff
@@ -180,6 +182,88 @@ impl Class {
...
+                max_angular_acceleration: 0.0,
+            },
+        }
+    }
+}
+
+/// Stats for a class of ship
+#[derive(Debug, Clone)]
+#[allow(missing_docs)]
+pub struct ClassStats {
```
Comment (rlane) [inline shared/api/src/lib.rs:260]: Let's plan on adding stats for weapons, radar, radio, etc to this class in the future. Given that, it wouldn't really make sense for this class to be used as "base_stats" in Ship anymore. So I think we should copy out the values from this class to Ship instead.

Fortunately I think we can just make a function `ClassStats -> Ship` and use that in ship::fighter/etc so there shouldn't be too much boilerplate.

Comment (twof) [reply shared/api/src/lib.rs:260]: Oh yeah I see what you're saying. I'll make that change.

**Comment by twof** [comment]
@rlane  Alright I think we're good to go. Not sure if it's exactly what you meant, but I implemented `From<ClassStats>` on `ShipData`. I'm still pretty new to Rust, but that felt like it was a good way to do it. Let me know if you want any other changes!

**Comment by rlane** [comment]
Thanks!

**Pull request merged**